src/MessageContainer.js

```diff
@@ -109,13 +117,15 @@ export default class MessageContainer extends React.Component {
       message.user = {};
```

Comment:
Note that in the `renderRow` method there is a warning when message has not user. If the `message.fromSystem` concept is introduced, you need to review the warning condition.